WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -55,6 +59,10 @@ The following table summarizes the key differences between the channels:
 | Who can use | All customers through all channels | Software Assurance and cloud customers only |
 | Installation options | Server Core and Server with Desktop Experience | Server Core for container host and image and Nano Server container image |
 
+> [!IMPORTANT]
+> Please understand that the set of roles and features in Windows Server SAC, only available as Server Core installation option, differs from Windows Server LTSC, installed with Server Core installation option.
```

Comment:
codestyle/semantics: Remove a comma to preserve the effective meaning and add a missing definite article.
```suggestion
> Please understand that the set of roles and features in Windows Server SAC, only available as Server Core installation option, differs from Windows Server LTSC installed with the Server Core installation option.
```